gensim/models/word2vec.py

```diff
@@ -96,6 +96,7 @@
 from types import GeneratorType
 
 logger = logging.getLogger(__name__)
+logging.basicConfig()
```

Comment:
logging.basicConfig(format='%(asctime)s : %(levelname)s : %(message)s') is better